### ISSUE TITLE
Non-highlighted words trimmed

### ISSUE THREAD

**Issue by scriptcoded** [scriptcoded/sql-highlight#14]
**Describe the bug**
When highlighting a query that either begins or ends with one or more words that are not highlighted, those words will be trimmed.

**To Reproduce**
1. Run the following script:
   ```js
   const { highlight } = require('../index')

   console.log(highlight('SELECT id FROM users'))
   ```
2. Notice that the output is `SELECT id FROM`

**Expected behavior**
The output should be `SELECT id FROM users`

**Node.js:**
 - **Version**: v12.16.2

**Comment by jackpilowsky** [comment]
Run across this issue too. PR coming

**Comment by jackpilowsky** [comment]
PR submitted: https://github.com/scriptcoded/sql-highlight/pull/15

**Comment by scriptcoded** [comment]
Closed with #15
